TribalWars/src/WebApi/Controllers/TribeController.cs

```diff
@@ -29,13 +30,24 @@ public TribeController(IMapper mapper, ITribeService tribeService, IStringLocali
         [HttpPost("/create-tribe")]
         public async Task<IActionResult> CreateTribe([FromBody] TribeDto dto)
         {
-            var userId = new Guid();  //TODO: Read userId from session
+            var playerId = new Guid();  //TODO: Read playerId from session
             var tribe = _mapper.Map<Tribe>(dto);
 
-            var result = await _tribeService.CreateTribe(tribe, userId);
+            var result = await _tribeService.CreateTribe(tribe, playerId);
             return result.Succeeded 
                 ? Ok(new SuccessResponse(_localizer["AddTribeSuccess"])) 
                 : BadRequest(new ErrorsResponse(result.Errors));
         }
+
+        [HttpGet("/tribe-details")]
+        public async Task<ActionResult<TribeDetailsVM>> TribeDetails()
+        {
+            var playerId = new Guid();  //TODO: Read playerId from session
+
+            var result = await _tribeService.TribeDetails(playerId);
+            return result.Item1.Succeeded
+                ? Ok(result.Item2)
+                : BadRequest(new ErrorsResponse(result.Item1.Errors));
```

Comment:
Same as before, instead of result.Item1.Succeded, result.Item2 you could access result.Succeded, result.Content